src/index.tsx

```diff
@@ -81,8 +81,8 @@ const FastUrlModule: FastUrlNative = {
 export class URLSearchParams {
   private _urlSearchParams: ReturnType<FastUrlNative['URLSearchParams']>;
 
-  constructor(url: string) {
-    this._urlSearchParams = FastUrlModule.URLSearchParams(url);
+  constructor(url?: string) {
+    this._urlSearchParams = FastUrlModule.URLSearchParams(url ?? '');
```

Comment:
You can construct `URLSearchParams` without a url initially - it is done quite often when constructing a search string